content/en/blog/2025/ai-agent-observability/index.md

```diff
@@ -0,0 +1,216 @@
+standardize AI Agent observability. This effort is primarily driven by:
+
+- **Agent Semantic Convention** – A draft AI Agent Semantic Convention has
+  already been established and finalized as part of the discussions in the
+  [OpenTelemetry semantic conventions repository](https://github.com/open-telemetry/semantic-conventions/issues/1732).
+  The initial AI Agent Semantic Convention is based on
+  [Google's AI Agent white paper](https://www.kaggle.com/whitepaper-agents),
+  providing a foundational framework for defining observability standards.
+  Moving forward, we will continue to refine and enhance this initial convention
+  to make it more robust and comprehensive.
+- **Agent Framework Semantic Convention** – Now, the focus has shifted towards
+  defining a common semantic convention for all AI Agent frameworks. This effort
+  is being discussed in
+  [this OpenTelemetry issue](https://github.com/open-telemetry/semantic-conventions/issues/1530)
+  and aims to establish a standardized approach for frameworks such as IBM Bee
+  Stack, IBM wxFlow, CrewAI, AutoGen, LangGraph, and others. Additionally,
+  different AI Agent frameworks will be able to define their own Framework
+  Vendor Specific Semantic Convention while adhering to the common standard.
+
+By establishing these conventions, we ensure that AI Agent frameworks can report
+standardized metrics, traces, and logs, making it easier to integrate
+observability solutions and compare performance across different frameworks.
+
+Note: Experimental conventions already exist in OpenTelemetry for models at
+[GenAI Semantic Convention](/docs/specs/semconv/gen-ai/).
+
+### Instrumentation Approaches for AI Agent Frameworks
+
+In order to make a system observable, it must be instrumented: That is, code
+from the system’s components must
+[emit traces, metrics, and logs](/docs/concepts/instrumentation/).
+
+Different AI Agent frameworks have varying approaches to implementing
+observability, mainly categorized into two options:
+
+#### Option 1: Baked-in Instrumentation
+
+The first option is to implement built-in instrumentation that emits telemetry
+using OpenTelemetry semantic conventions. This means observability is a native
+feature, allowing users to seamlessly track agent performance, task execution,
+and resource utilization. Some AI Agent frameworks, such as CrewAI, follow this
+pattern.
+
+As a developer of an agent framework, here are some pros and cons of this
+baked-in instrumentation:
+
+- Pros
+  - You can take on the maintenance overhead of keeping the instrumentation for
+    telemetry up-to-date.
+  - Simplifies adoption for users unfamiliar with OpenTelemetry configuration.
+- Cons
+  - Adds bloat to the framework for users who do not need observability
+    features.
+  - Risk of version lock-in if the framework’s OpenTelemetry dependencies lag
+    behind upstream updates.
+  - Less flexibility for advanced users who prefer custom instrumentation.
```

Comment:
Is this actually true? I've not heard of baking in instrumentation making it difficult for users who also wish to augment autoinstrumentation with custom instrumentation, nor those who wish to just turn off autoinstrumentation and do it all themselves.